tests/testthat/test-legend.R

```diff
@@ -0,0 +1,52 @@
+context("legend")
+
+test_that("easy_remove_legend works with no argument", {
```

Comment:
I think this is the simplest way to go forward with the testing. The `gg` objects *should* be the same if we've done everything right (that's the point of the helpers).